multiapps-common/src/test/java/org/cloudfoundry/multiapps/common/util/JsonUtilTest.java

```diff
@@ -3,14 +3,34 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import java.io.InputStream;
+import java.nio.charset.Charset;
 import java.util.Collections;
 import java.util.Map;
 import java.util.TreeMap;
 
+import org.apache.commons.io.IOUtils;
+import org.cloudfoundry.multiapps.common.Messages;
+import org.cloudfoundry.multiapps.common.ParsingException;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 class JsonUtilTest {
 
+    @Test
+    void testConvertJsonToMapWithInvalidJsonInputStream() {
+        String invalidJson = "{]";
+        InputStream invalidJsonInputStream = IOUtils.toInputStream(invalidJson, Charset.defaultCharset());
+
+        ParsingException resultException = Assertions.assertThrows(ParsingException.class,
+                                                                   () -> JsonUtil.convertJsonToMap(invalidJsonInputStream));
+
+        String expectedMessage = Messages.CANNOT_CONVERT_JSON_STREAM_TO_MAP + ": " + resultException.getCause()
+                                                                                                    .getMessage();
+        ParsingException expectedException = new ParsingException(resultException.getCause(), expectedMessage, invalidJsonInputStream);
+        Assertions.assertEquals(expectedException.getMessage(), resultException.getMessage());
```

Comment:
It might be simpler to just do:
```
String exceptionMessage = resultException.getMessage();
assertTrue(exceptionMessage.contains(resultException.getCause().getMessage(), MessageFormat.format("Expected exception [{0}] to contain [{1}]!", exceptionMessage, resultException.getCause().getMessage()));
```